bleachbit/FileUtilities.py

```diff
@@ -924,12 +954,26 @@ def estimate_completion():
                 break
             else:
                 raise
+
+        # Get the file system type from the given path
+        fstype = get_filesystem_type(pathname)
+        fstype = fstype[0]
+        print('File System:' + fstype)
+        # print(f.name) # Added by Marvin for debugging #issue 1051
         last_idle = time.time()
         # Write large blocks to quickly fill the disk.
         blanks = b'\0' * 65536
+        writtensize = 0
+        
         while True:
             try:
-                f.write(blanks)
+                if fstype != 'vfat':
+                    f.write(blanks)
+                elif writtensize < 4 * 1024 * 1024 * 1024 - 65536:
```

Comment:
Please add a code comment to explain the significance of this magic number.

Reply:
In the ubuntu system, there should be a error if the size of file is 4GB, no error if the size of file is less than 4GB. When we wipe the path, since we are writing 65536 bytes at a time if the size of space left is large than 4GB, the temp file size would be 4GB in the FAT32 system. In this case, the system always give us EFBIG error. I thought we need to set the maximum size of temp file as 4GB-65536 bytes.

Reply:
I will add the comment for this magic number.